content/en/blog/_posts/2019-03-12-kubeedge-k8s-based-edge-intro.md

```diff
@@ -0,0 +1,114 @@
+
+**EdgeHub**: Communication interface module at Edge. It is a web socket client responsible for interacting with Cloud Service for the edge computing. 
+
+**CloudHub**: Communication interface module at Cloud. A web socket server responsible for watching changes at the cloud side, caching and sending messages to EdgeHub.
+
+**EdgeController**: Manages the edge nodes. It is an extended kubernetes controller which manages edge nodes and pods metadata so that the data can be targeted to a specific edge node.
+
+**EventBus**: Handles the internal edge communications using MQTT. It is an MQTT client to interact with MQTT servers (mosquitto), offering publish and subscribe capabilities to other components.
+
+**DeviceTwin**: It is software mirror for devices that handles the device metadata. This module helps in handling device status and syncing the same to cloud. It also provides query interfaces for applications, as it interfaces to a lightweight database (SQLite).
+
+**MetaManager**: It manages the metadata at edge node. This is the message processor between edged and edgehub. It is also responsible for storing/retrieving metadata to/from a lightweight database (SQLite).
+
+Even if want to add more control plane module based on the architecture refinement and improvement (for example enhanced security), it is simple as it uses consistent registration and modular communication within these modules.
+
+
+**<p align="center">KubeEdge provides scalable light weight Kubernetes Native Edge Computing Platform which can work in offline mode.</p>** 
+
+**<p align="center">It helps simplified edge application development and deployment.</p>**
+
+**<p align="center">Cloud vendor agnostic and can run the cloud core modules on any compute node.</p>**
+
+
+## Release 0.1 to 0.2 -- game changer!
+
+KubeEdge v0.1 was released by end of Dec, 2018 with very basic edge features to manage edge application along with kubernetes API primitives for node, pod, config etc. In ~2 months, KubeEdge came with the release v0.2 on 5th March, 2019. This release provides the cloud core modules and enables the end to end open source edge computing solution. The cloud core modules can be deployed to any compute node from any cloud vendors or on-prem.
+
+Now, the complete edge solution can be installed and tested very easily, also with a laptop.
+
+## Run Anywhere - Simple and Light
+
+As described, the KubeEdge Edge and Cloud core components can be deployed easily and can run the user applications. The edge core has a foot print of 66MB and just needs 30MB memory to run. Similarly the cloud core can run on any cloud nodes. (User can experience by running it on a laptop as well)
+
+The installation is simple and can be done in few steps:
+  1. Setup the pre-requisites Docker, Kubernetes, MQTT and openssl
+  2. Clone and Build KubeEdge Cloud and Edge
+  3. Run Cloud
+  4. Run Edge
+
+The detailed steps for each are available at https://github.com/kubeedge/kubeedge
+
```

Comment:
can we update link like: [KubeEdge/kubeedge](https://github.com/kubeedge/kubeedge) ?

Reply:
Fixed.